src/style.css

```diff
@@ -0,0 +1,144 @@
+
+/* #region PC container */
+.body {
+  margin: 0;
+}
+
+/* #endregion */
+
+/* #region card-info */
+.card-info {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: 200px;
+}
+
+.card-info__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card-info__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0;
+  padding-bottom: 4px;
+  padding-left: 16px;
+  padding-right: 16px;
```

Comment:
you can set horizontal paddings for whole card in one place instead of setting padding or margin for every element